domain/src/main/java/com/teamagam/gimelgimel/domain/layers/ProcessVectorLayersInteractor.java

```diff
@@ -38,6 +40,7 @@
   private final AlertedVectorLayerRepository mAlertedVectorLayerRepository;
   private final MessagesRepository mMessagesRepository;
   private final RetryWithDelay mRetryStrategy;
+  private AlertMessageTextCreator mAlertMessageTextCreator;
```

Comment:
Use the final keyword when possible.
This comment is written here, but you could've used it a couple of other times throughout this CR.
It helps us understand if the field has something to do with the object's state or not.

Reply:
done